grakn-core/src/main/java/ai/grakn/graql/Query.java

```diff
@@ -60,7 +60,7 @@
      * Execute the query and return a converted stream of results
      */
     @CheckReturnValue
-    <T> Stream<T> results(GraqlConverter<?, T> converter);
+    <S> Stream<S> results(GraqlConverter<?, S> converter);
```

Comment:
why T become S? is it because we are handling Streams here?

Reply:
No, it's more boring than that! We were already using `T` in this class (`Query<T>`), so it was confusing.